locales/en.default.schema.json

```diff
@@ -1927,7 +1927,8 @@
         },
         "enable_video_looping": {
           "label": "Enable video looping"
-        }
+        },
+        "section_colors_heading": "Colors"
```

Comment:
From a settings parity perspective, instead of making this apart of `"main-product"`, it would probably make more sense to live under `sections.all.colors` instead.

This way, if we need to add a "Colors" heading to other sections, we don't have to add an additional, `"section_colors_heading": "Colors"` under that section. Then in the section file, we can simply reference this heading the same way we reference the colors, something like this:

```json
{
  "type": "header",
  "content": "t:sections.all.colors.heading"
},
```

The heading can be added here: https://github.com/Shopify/dawn/blob/f538b0a97903173c0298d8b268bcfb130664670a/locales/en.default.schema.json#L368-L386

Reply:
We actually had a chat about it and since it's only used in one place for now thought it would make sense for it to live on the main-product section. 

But if we're doing it on featured product as well that would probably be a good idea 🤔

Reply:
>But if we're doing it on featured product as well that would probably be a good idea 🤔

I agree 👍🏼 

I was also going to say that the Collapsible content section could also probably use this "Colors" heading:

![image](https://user-images.githubusercontent.com/28404165/184021058-edf6f462-b0e2-42a0-84d4-c1c5cf7ecdce.png)

Reply:
I like the idea of moving "Colors" heading to `settings.all.colors` there is a chance that we will use it somewhere else.

I talked to @Oliviammarcello and there are a few unclear things about featured product yet. 

I totally agree about the Collapsible content section. I'd prefer to do it in a different PR though.